sips/mev.md

```diff
@@ -0,0 +1,91 @@
+| Author                    | Title                           | Category | Status              |
+| ------------------------- | ------------------------------- | -------- | ------------------- |
+| Moshe Revah (@moshe-blox) | Support externally built blocks | Core     | open-for-discussion |
+
+## Summary
+
+Support the production and proposal of blocks built by external entities, allowing operators to access blocks from a marketplace of builders such as [Flashbots](https://boost-relay.flashbots.net/).
+
+## Rational & Design Goals
+
+Block building is a computationally intensive task, and even more so when blockspace utilization and profit maximization are in mind. Additionally, many of the more profitable transactions, so-called MEV, are private and only accessible to validators through external block builders.
+
+Currently, operators wishing to offer competitive returns must be sophisticated block builders with private transaction flows. This SIP aims to change that by allowing them to register with and propose blocks from external builders who comply to [ethereum/builder-specs](https://github.com/ethereum/builder-specs).
+
+## Specification
+
+### Configuration
+
+Operators who wish to propose blinded blocks must configure their node to do so, otherwise it would only propose standard blocks and reject QBFT proposals containing blinded blocks.
+
+Therefore, validators should solely select operators with matching configuration, otherwise consensus might be unreachable.
```

Comment:
what is the motivation for disabling support?
Why not just always have it on?

If the beacon node will want to create a non-blinded block then it is fine. Else it is also fine.

Btw, the used relay is a configuration of the beacon node only?

Reply:
Beacon nodes would always return a blinded block if you asked for it. If it can't source it externally, it would build it locally.

I see where the confusion is, committed this to clarify:
> Operators who wish to propose blinded blocks must configure their **SSV** node to do so

Reply:
I meant that maybe you add an explanation like you did here https://github.com/bloxapp/SIPs/pull/25#discussion_r1196242791.

If not in the SIP I think it should be written somewhere accessible.